### PR TITLE
Hotfix/sharding/perf test

### PR DESCRIPTION
**Pull request by liayoo**
- Added LIGHTWEIGHT flag for running nodes in the lightweight mode
- EPOCH_MS: 1s -> 3s
- Fixed cleanUpForNewBlock() to check last_votes transactions
- Fixed consensus bugs
- Updated and created deploy & reset scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can override default port numbering system by setting `PORT` and `P2P_PORT`
 - Deploy code (in common with Node server)
 ```
 gcloud init
-sh deploy_gcp.sh {dev|spring|summer} <YOUR_GCP_USER_NAME>
+sh deploy_gcp.sh {dev|spring|summer} <YOUR_GCP_USER_NAME> <NUMBER_OF_SHARDS>
```
Comment (platfowner) [inline README.md:34]: Which value should be set used the last parameter for parent blockchain? Could you add some comments?

File: deploy_shard_clusters_gcp.sh [hunk 2]
```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
```
Comment (platfowner) [inline deploy_shard_clusters_gcp.sh:1]: As discussed offline, this file contents can replace deploy_gcp.sh.

File: consensus/index.js [hunk 3]
```diff
@@ -325,6 +344,9 @@ class Consensus {
         }
       }, false);
     }
+    if (LIGHTWEIGHT) {
```
Comment (platfowner) [inline consensus/index.js:347]: Could you add some comments why it's enabled only when LIGHTWEIGHT = true?

File: README.md (hunk 1 above)
Comment (liayoo) [reply README.md:35]: Thanks, done!

File: deploy_shard_clusters_gcp.sh (hunk 2 above)
Comment (liayoo) [reply deploy_shard_clusters_gcp.sh]: Thanks! Replaced the contents and removed this file.

File: consensus/index.js (hunk 3 above)
Comment (liayoo) [reply consensus/index.js:348]: Done!

**Comment by liayoo** [comment]
Addressed all comments, PTAL.

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by liayoo** [comment]
Thanks for the quick review! Now merging..

**Pull request merged**